QRCoderApiTests/net35+net40+net50+net50-windows+netstandard20/QRCoder.approved.txt

```diff
@@ -895,6 +895,7 @@ namespace QRCoder
         public static QRCoder.QRCodeData GenerateQrCode(string plainText, QRCoder.QRCodeGenerator.ECCLevel eccLevel, bool forceUtf8 = false, bool utf8BOM = false, QRCoder.QRCodeGenerator.EciMode eciMode = 0, int requestedVersion = -1) { }
         public enum ECCLevel
         {
+            Default = -1,
             L = 0,
             M = 1,
             Q = 2,
```

Comment:
Only API change is to add a new enum entry, ensuring compatibility with code built against prior versions.